### ISSUE TITLE
empty-line-between-blocks fails on child block

### ISSUE THREAD

**Issue by attiks** [sasstools/sass-lint#61]
Given the following scss file

```scss
.selector {
  color: $header;

  @include breakpoint($tab1) {
    &:after {
      content: '|';
      padding-left: 12px;
    }
  }
}
```

I get 

```
test.scss
  5:4  warning  Space expected between blocks  empty-line-between-blocks
```

Is this the correct behavior? I tought it should be ok?

**Comment by mherchel** [comment]
Just a clarification (I'm running into this now while trying to migrate from scsslint to sasslint).

If the parent selector has no rules, and has an immediate child selector without an empty line in between, the test will *pass* on [scsslint](https://github.com/brigade/scss-lint), but fail with sasslint.

Another example: This is allowed by scsslint, but not by sasslint
````scss
.cta-row-equal-height {
  .cta-wrapper {
    border: 15px solid $color-white;
    border-top-width: 0;
    @media #{$medium-up} {
      border-top-width: 15px;
    }
  }
````
